src/test/java/ibm/jceplus/junit/base/BaseTestAES.java

```diff
@@ -27,44 +26,42 @@
 import javax.crypto.spec.RC2ParameterSpec;
 import javax.crypto.spec.RC5ParameterSpec;
 import javax.crypto.spec.SecretKeySpec;
-import org.junit.Assume;
-import org.junit.Before;
-import org.junit.BeforeClass;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
 public class BaseTestAES extends BaseTestCipher {
 
     // 14 bytes: PASSED
-    static final byte[] plainText14 = "12345678123456".getBytes();
```

Comment:
I put static back into this set of declarations. I was removing many static declarations for tests that are multithreaded. However since this is a `final` declaration i think this is prob ok to put back in.